.github/workflows/build-release.yml

```diff
@@ -0,0 +1,42 @@
+on:
+  push:
+    branches:
+      - master
+
+name: build-release
+
+jobs:
+  build:
+    name: build-release
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Code
+        uses: actions/checkout@master
+
+      - name: Setup Node.js
+        uses: actions/setup-node@v1
+        with:
+          node-version: "12.x"
+
+      - run: sudo npm i -g @vercel/ncc
+
+      - run: npm install 
+
+      - name: Compile files
+        run: |
+          ncc build -C -m src/main.ts
```

Comment:
Can you please also create a section in README to tell how to build the code. This will help anyone trying to update the code in future

Reply:
yes, have documented it as .md file
PR : https://github.com/Azure/cli/pull/27